control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -90,6 +95,9 @@ type HostedControlPlaneReconciler struct {
 	// ManagementClusterCapabilities can be asked for support of optional management cluster capabilities
 	ManagementClusterCapabilities *capabilities.ManagementClusterCapabilities
 
+	// SetSecurityContext is used to configure Security Context for containers
+	// SetSecurityContext bool
```

Comment:
why is this commented code?

Reply:
Fixed to use to this struct. https://github.com/openshift/hypershift/pull/846/commits/17d7cd45a2a197b4478808bf2d1cfbd0fdb9a8a2